metagraph/src/common/vectors/vector_algorithm.hpp

```diff
@@ -489,6 +676,49 @@ prev_bit(const t_int_vec &v,
 }
 
 
+// Return an int_vector whose limbs are aligned to alignment bytes.
+// This is useful for algorithms requiring access to larger words in the underlying
+// vector (atomic increment, SIMD, etc.)
+template <int width = 0>
+inline sdsl::int_vector<width> aligned_int_vector(size_t size = 0, uint64_t val = 0,
+                                                  uint8_t var_width = 0,
+                                                  size_t alignment = 8) {
```

Comment:
does it make sense to specify if this are bits or bytes?

Reply:
Usually alignment is specified in bytes